UI/reconciliation/report.html

```diff
@@ -293,7 +342,7 @@
     name = "action",
     text = text('Approve'),
     value = 'approve',
-    disabled = submit_allowed == 0 ? 'false' : 'true',
+    disabled = out_of_balance == '0.00' ? 'false' : 'true',
```

Comment:
This is a string comparison, but what happens when the number of digits is set to 3 (or, in case of Japanese Yen, to zero)?